sycl/test-e2e/AddressSanitizer/out-of-bounds/USM/matrix_multiply.cpp

```diff
@@ -0,0 +1,88 @@
+// REQUIRES: linux
+// RUN: %{build} %device_sanitizer_flags -DMALLOC_DEVICE -O1 -g -o %t
+// RUN: %{run} not %t &> %t.txt ; FileCheck --check-prefixes CHECK,CHECK-DEVICE --input-file %t.txt %s
```

Comment:
I think the temporary txt is not needed now?

Reply:
IMO, temporary file is very helpful to find out why FileCheck failed, so I keep it intentionally.

Reply:
JFYI: `FILECHECK_OPTS="--dump-input=always"`

Reply:
> JFYI: `FILECHECK_OPTS="--dump-input=always"`

Thanks for you tips, I'll try it